packages/js/test/plugins/auctionHouseModule/createAuctionHouse.test.ts

```diff
@@ -143,3 +150,84 @@ test('[auctionHouseModule] create new Auction House with SPL treasury', async (t
     },
   });
 });
+
+test('[auctionHouseModule] create new Auctioneer Auction House', async (t: Test) => {
+  // Given we have a Metaplex instance.
+  const mx = await metaplex();
+
+  const auctioneerAuthority = Keypair.generate();
+
+  // When we create a new Auctioneer Auction House.
+  const { auctionHouse } = await mx
+    .auctions()
+    .createAuctionHouse({
+      sellerFeeBasisPoints: 200,
+      auctioneerAuthority: auctioneerAuthority.publicKey,
+    })
+    .run();
+
+  // Then the new Auction House has Auctioneer attached.
+  t.ok(auctionHouse.hasAuctioneer);
```

Comment:
When the `AuctionHouse` model also has info on `auctioneerAuthority` and `auctioneerScopes`, let's also make assertions on them. We should also have another test that does the same as this test but by providing some scopes explicitly.